app/views/shared/_footer.html.erb

```diff
@@ -29,14 +28,13 @@
       <div class="col-md-4 col-xs-5">
         <div class="text-right">
           <p>
-            <img class='social-logo' src='<%= image_url('insta.png')%>' alt="Instagram">
-            <img class='social-logo' src='<%= image_url('facebook.png')%>' alt="Facebook">
-            <img class='social-logo' src='<%= image_url('twitter.jpg')%>' alt="Twitter">
+            <a href="https://twitter.com/oregondigcol"><img class='social-logo' src='<%= image_url('twitter.jpg')%>' alt="Twitter"></a>
+            <a class="footer-link-text" href="https://twitter.com/oregondigcol">@oregondigcol</a>
```

Comment:
If the image and text are in the same a-tag then you can blank the alt text on the image (alt="") and it makes tabbing easier